test/modules/XMLHttpRequest/response/xhr-response-without-body.test.ts

```diff
@@ -0,0 +1,108 @@
+// @vitest-environment jsdom
+import { afterAll, afterEach, beforeAll, expect, it, vi } from 'vitest'
+import { HttpServer } from '@open-draft/test-server/http'
+import { XMLHttpRequestInterceptor } from '../../../../src/interceptors/XMLHttpRequest'
+import { createXMLHttpRequest, useCors } from '../../../helpers'
+import type { HttpRequestEventMap } from '../../../../src'
+
+const httpServer = new HttpServer((app) => {
+  app.use(useCors)
+  app.get('/:statusCode', (req, res) =>
```

Comment:
I made this server route return whichever response status code that was requested by the client.

Reply:
This is a great approach. I didn't think of it right away.

Reply:
I liked how you orchestrated the test! This is but a small adjustment.